frontend/test/readme.md

```diff
@@ -0,0 +1,16 @@
+installation
+===================================
+```
+pip install selenium
```

Comment:
put this in the requirement.txt, so that it will get installed every time by default